tests.py

```
@@ -0,0 +1,35 @@
+import unittest
+from fibo import fibonacci, geometric_series_aprox
```

Ruff

`fibo.geometric_series_aprox` imported but unused

Remove unused import: `fibo.geometric_series_aprox`

(F401)